skpro/domains/domains.py

```diff
@@ -126,6 +126,13 @@ def boundary(self) -> Tuple[float, float]:
         """Return the boundary of the interval, i.e., the extremities."""
         return self._left, self._right
 
+    def get_test_params(cls, parameter_set="default"):
```

Comment:
I think you are missing the `@classmethod` decorator, then it should work.

Reply:
added

Reply:
restarted tests